corehq/apps/reports/standard/monitoring.py

```diff
@@ -779,69 +781,54 @@ def to_minutes_raw(val_in_s):
                 s=seconds,
             )
 
-        def _fmt(pretty_fn, val):
-            return format_datatables_data(pretty_fn(val), val)
-
         def _fmt_ts(timestamp):
             return format_datatables_data(to_minutes(timestamp), timestamp, to_minutes_raw(timestamp))
 
-        def get_data(users, group_by_user=True):
-            query = FormData.objects.filter(xmlns=self.selected_form_data['xmlns'])
-
-            date_field = 'received_on' if self.by_submission_time else 'time_end'
-            date_filter = {
-                '{}__range'.format(date_field): (self.datespan.startdate_utc, self.datespan.enddate_utc)
-            }
-            query = query.filter(**date_filter)
-
-            if users:
-                query = query.filter(user_id__in=users)
-
-            if self.selected_form_data['app_id'] is not None:
-                query = query.filter(app_id=self.selected_form_data['app_id'])
-
-            if group_by_user:
-                query = query.values('user_id')
-                return query.annotate(Max('duration')) \
-                    .annotate(Min('duration')) \
-                    .annotate(Avg('duration')) \
-                    .annotate(StdDev('duration')) \
-                    .annotate(Count('duration'))
-            else:
-                return query.aggregate(
-                    Max('duration'),
-                    Min('duration'),
-                    Avg('duration'),
-                    StdDev('duration'),
-                    Count('duration')
-                )
-
         mobile_user_and_group_slugs = self.request.GET.getlist(EMWF.slug)
         users_data = EMWF.pull_users_and_groups(
             self.domain,
             mobile_user_and_group_slugs,
         )
         user_ids = [user.user_id for user in users_data.combined_users]
+        app_id = self.selected_form_data['app_id']
```

Comment:
originally this allowed for the app_id to be `None`, i don't see any scenario where that would happen. do you know if i'm overlooking something?